modules/ownKeys.js

```diff
@@ -0,0 +1,16 @@
+import objectKeys from 'object-keys'
+
+const ownKeys = (object) => {
+  if (Reflect && Reflect.ownKeys)
+    return Reflect.ownKeys(object)
+
+  const keys = objectKeys(object)
+
+  if (Object.getOwnPropertySymbols)
+    return keys.concat(Object.getOwnPropertySymbols(object))
```

Comment:
Thanks! It'd be cool if you added a test that included both enumerable and non-enumerable string and Symbol keys.